<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1110</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>16</width>
      <height>391</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout"/>
   </widget>
   <widget class="QPushButton" name="butZoomIn">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>butZoomIn</string>
    </property>
   </widget>
   <widget class="QPushButton" name="butZoomOut">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>butZoomOut</string>
    </property>
   </widget>
   <widget class="QPushButton" name="butFindWay">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>180</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>findWay</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>301</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="LineName">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>60</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>121</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QRadioButton" name="radioButton_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>46</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>精确模式</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>80</y>
       <width>51</width>
       <height>42</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="timeMinute">
      <item>
       <widget class="QLabel" name="minute">
        <property name="text">
         <string>分</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="inputTime_2"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>80</y>
       <width>51</width>
       <height>42</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="timeHour">
      <item>
       <widget class="QLabel" name="hour">
        <property name="text">
         <string>时</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="inputTime"/>
      </item>
     </layout>
    </widget>
    <widget class="QRadioButton" name="radioButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>普通模式</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="ordest" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>211</width>
      <height>151</height>
     </rect>
    </property>
    <widget class="QLineEdit" name="stationOrigin">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>60</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="stationDestination">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>100</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="originNote">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>60</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="destinationNote">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>100</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1110</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
